# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/xfce4-terminal/login-shell.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- xfce4-terminal-1.1.5/terminal/terminal-preferences.c.vanilla	2025-12-04 18:39:15.577536777 +0200
+++ xfce4-terminal-1.1.5/terminal/terminal-preferences.c	2025-12-04 18:39:36.013426080 +0200
@@ -580,7 +580,7 @@
     g_param_spec_boolean ("command-login-shell",
                           NULL,
                           "CommandLoginShell",
-                          FALSE,
+                          TRUE,
                           G_PARAM_READWRITE | G_PARAM_STATIC_STRINGS);
 
   /**
